terraform/executor/Dockerfile-0.12-slim

```diff
@@ -0,0 +1,88 @@
+FROM golang:1.12.6 AS tfmask
+
+RUN git clone https://github.com/cloudposse/tfmask.git
+RUN cd tfmask && make && make go/build
+
+FROM debian:buster-slim
+
+ARG TF_VERSIONS="0.12.29"
+ARG DEFAULT_TF_VERSION=0.12.29
+ARG GCLOUD_VERSION=279.0.0
+ARG AWSCLI_VERSION=1.17.11
+ARG HELM2_VERSION=2.16.9
+ARG HELM3_VERSION=3.0.3
+ARG TFSWITCH_VERSION=0.8.832
+ARG TF_AIVEN_VERSIONS="1.1.4"
+ARG TF_OVO_VERSIONS="1.0.0"
+
+# Terraform environment variables
+ENV CHECKPOINT_DISABLE=true
+ENV TF_IN_AUTOMATION=yep
+ENV TF_INPUT=false
+ENV TF_PLUGIN_CACHE_DIR=/usr/local/share/terraform/plugin-cache
+ENV CLOUDSDK_PYTHON=python3
+
+RUN apt-get update && apt-get install -y \
+    git \
+    ssh \
+    tar \
+    gzip \
+    ca-certificates \
+    curl \
+    unzip \
+    jq \
+    python3 \
+    python3-requests \
+    python3-pip \
+    wget \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsL https://github.com/warrensbox/terraform-switcher/releases/download/${TFSWITCH_VERSION}/terraform-switcher_${TFSWITCH_VERSION}_linux_amd64.tar.gz -o tfswitch.tar.gz \
+ && tar -xvf tfswitch.tar.gz \
+ && mv tfswitch /usr/local/bin \
+ && rm -rf tfswitch \
```

Comment:
I think you'll also want to delete `tfswitch.tar.gz` to save a few more bytes (~9M).

Reply:
Correction: this line should be `tfswitch.tar.gz`.

Reply:
looks like it should be
```
rm -rf tfswitch.tar.gz
```
rather than `tfswitch`